backend/app/src/main/java/ch/zhaw/studyflow/domain/appointment/SQLAppointmentDao.java

```diff
@@ -0,0 +1,107 @@
+package ch.zhaw.studyflow.domain.appointment;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * SQL implementation of the AppointmentDao interface.
+ */
+public class SQLAppointmentDao implements AppointmentDao {
+    private final Connection connection;
+
+    public SQLAppointmentDao(Connection connection) {
+        this.connection = connection;
+    }
+
+    @Override
+    public Appointment create(Appointment appointment) {
+        String sql = "INSERT INTO appointments (id, start_time, end_time, calendar_id) VALUES (?, ?, ?, ?)";
+        try (PreparedStatement stmt = connection.prepareStatement(sql)) {
+            stmt.setLong(1, appointment.getId());
```

Comment:
The user shouldn't set an ID when creating an appointment. The ID is given to the appointment by the database (auto-increment).